### PR TITLE
[CORL-1476] Community Username Breaking

### PR DESCRIPTION
**Pull request by wyattjoh**
## What does this PR do?

Breaks long usernames in the community tab.

## What changes to the GraphQL/Database Schema does this PR introduce?

None.

## How do I test this PR?

Have a long username such as:

```
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua```

And see that it does not change the column widths.

### CODE REVIEW

**Review by tessalt — APPROVED** [review]
👍 same issue had same solution stream-side

**Review by nick-funk — APPROVED** [review]

**Review by alinhle — CHANGES_REQUESTED** [review]
wait :-D

File: src/core/client/admin/routes/Community/UserRow.css [hunk 1]
```diff
@@ -26,6 +26,9 @@
   font-weight: var(--font-weight-primary-semi-bold);
   font-size: var(--font-size-2);
   line-height: 1.14;
+  line-break: anywhere;
```
Comment (alinhle) [inline src/core/client/admin/routes/Community/UserRow.css]:
```suggestion
```

Comment (alinhle) [reply src/core/client/admin/routes/Community/UserRow.css]: Remove this line above, as it is too aggressive: 
image

**Review by alinhle — CHANGES_REQUESTED** [review]

**Review by alinhle — APPROVED** [review]

**Pull request merged**